mentat/commands.py

```diff
@@ -390,3 +388,18 @@ def argument_names(cls) -> list[str]:
     @classmethod
     def help_message(cls) -> str:
         return "Set a configuration option or omit value to see current value."
+
+
+class RunCommand(Command, command_name="run"):
```

Comment:
Another thing about our commands: we should probably split this file up and put each command in its own file. But we should do that refactor in a later pr.

Reply:
Agreed. I'm planning on doing a major refactor soon that breaks up our main folder into a lot of subfolders.